NetSQS/BatchResponse.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Linq;
+using Amazon.SQS.Model;
+
+namespace NetSQS
+{
+    /// <summary>
+    /// The result from sending a batch of messages to a queue. 
+    /// </summary>
+    public struct BatchResponse
```

Comment:
Should this and the other structs be classes based of the Microsoft blog post we read?